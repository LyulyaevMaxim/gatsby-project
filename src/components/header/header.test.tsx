import React from 'react'
import renderer from 'react-test-renderer'
import { PureHeader } from './index'

// TODO: https://react.i18next.com/misc/testing
describe('Header', () => {
  it('renders correctly', () => {
    expect(
      renderer
        .create(
          <PureHeader
            data={{
              site: {
                siteMetadata: {
                  title: 'Gatsby Project',
                },
                buildTime: '01-01-2021'
              },
              desktop: {},
            }}
          />
        )
        .toJSON()
    ).toMatchSnapshot()
  })
})
