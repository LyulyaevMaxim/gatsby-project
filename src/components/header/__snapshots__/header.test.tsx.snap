// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionFixed makeStyles-header-1 MuiAppBar-colorPrimary mui-fixed"
  data-test-id="header"
  style={
    Object {
      "visibility": undefined,
    }
  }
>
  <h6
    className="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
  >
    Gatsby Project
  </h6>
  <button
    className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-2"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiIconButton-label"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
        />
      </svg>
    </span>
  </button>
</header>
`;
